src/button/constants.js

```diff
@@ -20,7 +20,9 @@ export const SHAPE = {
 };
 
 export const SIZE = {
+  medium: 'default',
```

Comment:
We shouldn't introduce a new value that just refrences the `default`. Also, it's ok to have different naming from the designers' ones. Considering they change it pretty often, introduce new or deprecate older ones, we won't be able to follow it without breaking changes.

Reply:
I think ideally we should use the same naming to reduce friction

Reply:
it doesn't need to be included right now. so i'm going to remove so that this can land without deliberation

Reply:
```suggestion
```